server/container_create_linux.go

```diff
@@ -416,6 +470,9 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 			}
 		}
 	}
+	if err := s.addUnifiedResources(specgen, sb, containerConfig); err != nil {
+		return nil, err
```

Comment:
I'm not sure this fits in with `finalizeUserMapping`. This seems like something that should be in `pkg/container/container.go`

Reply:
good hint!  I'll move it